lib/post.rb

```diff
@@ -0,0 +1,35 @@
+require 'pg'
+
+class Post
+  attr_reader :id, :date, :author, :message
+
+  def initialize(id:, date:, author:, message:)
+    @id = id
+    @date = date
+    @author = author
+    @message = message
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+    result = connection.exec("SELECT * FROM peeps ORDER BY date DESC")
```

Comment:
I'm surprised queries without `;` work. I guess the framework adds the `;`. I would still make sure to add it however.